JeMPI_Apps/JeMPI_UI/Dockerfile

```diff
@@ -36,10 +36,24 @@ RUN yarn build
 
 FROM node:18-alpine as production-stage
 
+ENV BUILD_DEPS="gettext"  \
+    RUNTIME_DEPS="libintl"
+
+RUN set -x && \
+    apk add --update $RUNTIME_DEPS && \
+    apk add --virtual build_deps $BUILD_DEPS &&  \
+    cp /usr/bin/envsubst /usr/local/bin/envsubst && \
```

Comment:
Why do we need to install, copy and delete. Can't we just do `apk add getext`. i.e It there any specific reason you are moving it to `/usr./local/bin/envsubst`